Add abandoned log status (#945)
farmOS/farmOS: docs/guide/index.md

Asset details popup.

Below the map you will see "Upcoming tasks" and "Late tasks" on the left, which
summarize Logs that have a "pending" status. From here you can quickly select

> Logs and mark them as "done", reschedule them, clone them, or delete them.

Reviewer: Nit: I wonder if this can be more succinct and also mention the abandoned status:

```suggestion
Logs to rescheduled, cloned, deleted, or marked as "done"/"abandoned".
```

Author: That section refers to the "Upcoming tasks" and "Late tasks" on the dashboard and the bulk actions available on them. We don't provide a bulk action for "abandoning" logs in this PR. That might be a good idea for a followup, and at that point we could adjust this text IMO.
